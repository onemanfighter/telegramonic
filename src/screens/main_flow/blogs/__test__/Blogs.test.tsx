import { render } from '@testing-library/react';
import Blogs from '../Blogs';

describe('Blogs screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Blogs />);

    expect(container).toMatchSnapshot();
  });
});
