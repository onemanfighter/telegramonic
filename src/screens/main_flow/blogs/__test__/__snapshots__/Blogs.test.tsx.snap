// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blogs screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Blogs
    </h1>
  </div>
</div>
`;
